tutorials/fundamentals/shift.md

```diff
@@ -0,0 +1,9 @@
+The `shift()` function removes the first element from an array and returns the removed element.
+As a result, the length of the array changes as well.
+If the array is empty, `shift()` will return undefined.
```

Comment:
Move this sentence to a separate paragraph below the first example. And add an example of calling `shift()` on an empty array.